dev_guide/application_health.adoc

```diff
@@ -11,31 +11,93 @@ toc::[]
 
 == Overview
 
-In software systems, components can become unhealthy due to transient issues (temporary
-connectivity loss), configuration errors, or problems with external dependencies.  Applications
-running in OpenShift have a number of options to detect and handle unhealthy containers:
+In software systems, components can become unhealthy due to transient issues
+(such as temporary connectivity loss), configuration errors, or problems with
+external dependencies. OpenShift applications have a number of options to detect
+and handle unhealthy containers.
 
-<1> Container-level liveness checks
-<2> High-level application health checks
+[[container-health-checks-using-probes]]
 
-== Liveness checks
+== Container Health Checks Using Probes
 
-Each component in a distributed system has a notion of liveness scoped to that component.  Liveness
-is the quality of being correctly configured and operational on an individual level.  To express
-this, OpenShift offers a number of methods to establish container liveness and policies that
-determine how 'dead' containers are handled.
+A probe is a Kuberbetes action that periodically performs diagnostics on a
+running container. Currently, two types of probes exist, each serving a
+different purpose. Both probes can be configured in
+http://kubernetes.io/v1.0/docs/user-guide/liveness/[two ways]: HTTP checks and
+container execution checks.
```

Comment:
This states there are two ways (HTTP checks, container exec), but the k8s doc link below and https://github.com/openshift/origin/blob/master/examples/deployment/README.md#readiness-checks says there are three (includes TCP connection checks).